### ISSUE TITLE
reportbot patched?

### ISSUE THREAD

**Issue by qp1990** [Askwrite/node-csgo-reportbot#49]
I have this error;

{ job_id_source: '18446744073709551615',
job_id_target: '18446744073709551615',
eresult: 2,
gc_msg_src: 'GCProtoBufMsgSrc_Unspecified' } }

**Comment by COLATHOS** [comment]
I think that it means that it does work (same script than Zonerbot ou Husobot research). Fine for you. 
At least, you need 11 CSGO non-banned accounts to be efficient and to ban cheaters

**Comment by qp1990** [comment]
@COLATHOS Not working, I have 50 acc and the cheater not is banned

**Comment by Ar1i** [comment]
Then the cheater was not obvious enough.

**Comment by qp1990** [comment]
@Ar1i He used spinbot, you are a genius in this you know remove that error? a regard brother

**Comment by COLATHOS** [comment]
@qp1990 

x50 accounts ? Congrats for you, bro. How do you perform to get 50 accounts ? All not banned ? Your trick to get so many accs is amazing. Did you paid for each of them ? Which site ? Thanks

For sure, if this guy used spinbot so blatantly, and often, the gulty is so obvious to prove! May be another reason not to get banned: maybe this guy plays only casual now

Anyway, Bansystem and ZonerBot seem to work: they both use the same node process, more or less.
I had a look recently > approx 15-20 accounts get banned OW per day in these too sites. Not too bad ... but not free !

PS: Free csgoreports site is no longer available. Last banned account blocked on November,27 .... I dont know why.

**Comment by qp1990** [comment]
@COLATHOS 50 acc key asia, no banned, all in perfect condition

**Comment by COLATHOS** [comment]
@qp1990: You bought via G2A.com ? Never used that site. via Google: If i understand well, you Can't play CSGO with these asian accounts. But enough to load "accounts.txt" with username:password for reporbots. Correct ? Which other use ? Thanks

**Comment by qp1990** [comment]
it has nothing to do, I verify them by china vpn and ready and now I get that same error, before I went to zoner or any payment reportbot and fixed that error

**Comment by qp1990** [comment]
@COLATHOS it has nothing to do, I verify them by china vpn and ready and now I get that same error, before I went to zoner or any payment reportbot and fixed that error

**Comment by qp1990** [comment]
@COLATHOS I've been using nodebot since it came out

**Comment by COLATHOS** [comment]
OK, I had a look on some videos about G2A and how to get (cheap) asian STEAM+CSGO account by VPN (and fully playable for CSGO in western countries)

Simple question: Does npm work fine to report if I use 11 X asian accounts (included in "accounts.txt" by username:Password) ??

**Comment by qp1990** [comment]
@COLATHOS Yes, all the people who have made their own report bot buy csgo key asia accounts but .. When updated csgo gives that error i'm saying when I did not leave before

**Comment by COLATHOS** [comment]
Did you use process to get some "Free steam accounts" or did you pay for all your 50 x asian accounts (approx 50 x 5€ = 250€ ..... expensive indeed) ? 
Or another way to get 11 x (asian) accounts to "feed" the accounts.text ?

**Comment by qp1990** [comment]
@COLATHOS I pay 50, money is not the important thing but how to remove that error

**Comment by ilovegame1985** [comment]
Use this 
https://github.com/BeepIsla/csgo-commend-bot
